packages/core/src/lib/TableNew/atoms/TableBase.tsx

```diff
@@ -0,0 +1,161 @@
+    customNoDataText,
+    header,
+    isLoading,
+    noPagination,
+    pageNum,
+    pageSize,
+    tableData,
+    tableBackgroundColor,
+    isScrollableOnOverflow,
+    customTableBorder,
+    alignCellItems,
+    justifyCellItems,
+    cellPadding,
+    headerBgColor,
+    headerTextColor,
+    hover,
+    hoverBackgroundColor,
+    sortField,
+    order,
+    handleSortingChange,
+    onRowClick,
+}) => {
+    const computeCurrentData = (): (string | React.ReactNode)[][] => {
+        if (noPagination) {
+            return tableData;
+        }
+        const from = pageNum * pageSize;
+        const to = from + pageSize;
+        return tableData?.slice(from, to);
+    };
+
+    return (
+        <TableContainer
+            isScrollableOnOverflow={isScrollableOnOverflow}
+            customTableBorder={customTableBorder}
+        >
+            <TableStyled
+                tableBackgroundColor={tableBackgroundColor}
+                alignCellItems={alignCellItems}
+                headerTextColor={headerTextColor}
+                headerBgColor={headerBgColor}
+                hoverBackgroundColor={hoverBackgroundColor}
+            >
+                <thead>
+                    <tr>
+                        {header.map((head, key) => (
+                            <th key={`tableHeader_${key}`}>
+                                <DivTableCell
+                                    justifyCellItems={justifyCellItems}
+                                    cellPadding={cellPadding}
+                                    onClick={() => handleSortingChange(key)}
+                                    role="table-header"
+                                >
+                                    {head}
+                                    {sortField === key &&
+                                        (order === 'asc' ? (
+                                            <TriangleUp
+                                                title="triangle up icon"
+                                                titleId="table triangle up icon"
+                                                fill={color.blueGray50}
+                                            />
+                                        ) : (
+                                            <TriangleDown
+                                                title="triangle down icon"
+                                                titleId="table triangle down icon"
+                                                fill={color.blueGray50}
+                                            />
+                                        ))}
+                                </DivTableCell>
+                            </th>
+                        ))}
+                    </tr>
+                </thead>
+                {!isLoading && (
+                    <tbody>
+                        {computeCurrentData().map((row, rowKey) => (
+                            <tr className={`${hover && 'hover'}`}>
```

Comment:
please add a proper className to this component eg: `web3uikit-table-row`
This would give us the ability to customize outside of the kit too (might be needed for some complex use-case in admin)

Also we can remove `hover` prop.
Only `hoverBackgroundColor` should be ok. If hoverBackgroundColor has value we show that color else the default one.
What do you think ?